### PR TITLE
Add growable and resizable buffers

### PR DESCRIPTION
**Pull request by Elchi3**
Is this how it works? :) 

How do I create dist files? 
`npm run dist` didn't create anything.

### CODE REVIEW

**Comment by captainbrosset** [comment]
@Elchi3 I always give the name of the feature I want to refresh: `npm run dist features/xyz.yaml`

**Comment by Elchi3** [comment]
@captainbrosset ah great! Should have read that script more carefully. I guess I would expect all dists to be created/refreshed when not providing a feature name.

**Comment by Elchi3** [comment]
@ddbeck the action failed with a permission denied: https://github.com/web-platform-dx/web-features/actions/runs/9501887192/job/26188538265

**Comment by ddbeck** [comment]
@Elchi3 thank you! I understand the problem. I need split the labeling part of the workflow from the part that commits. I'll see if I can't fix this tomorrow.

**Comment by ddbeck** [comment]
I opened https://github.com/web-platform-dx/web-features/pull/1252/ to fix the permissions issue.

File: features/growable-sharedarraybuffer.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Growable SharedArrayBuffer
+description: ShareArrayBuffer objects can take a maximum length allowing them to grow in place.
```
Comment (ddbeck) [inline features/growable-sharedarraybuffer.yml:2]: The description here is a bit circular ("What's a growable buffer?" "A buffer that grows."). What do you think of something like this?

```suggestion
description: The `grow()` method of a `SharedArrayBuffer`, constructed with the `maxByteLength` option, increases the size of the buffer in place.
```

File: features/resizable-arraybuffer.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name: Resizable ArrayBuffer
+description: ArrayBuffer objects can take a maximum length allowing them to grow and shrink in place.
```
Comment (ddbeck) [inline features/resizable-arraybuffer.yml:2]: For symmetry:

```suggestion
description: The `resize()` method of an `ArrayBuffer`, constructed with the `maxByteLength` option, changes the size of the buffer in place.
```

**Review by ddbeck — APPROVED** [review]

**Pull request merged**